README.md

```diff
@@ -236,31 +236,29 @@ Example usage to anonymize FHIR resource files in a folder:
 
 ## Configuration file format
 
-The configuration is specified in JSON format. It has three high-level sections. Two of these sections, namely _pathRules_, and _typeRules_ are meant to specify de-identification methods for data elements. De-identification configuration specified in the _pathRules_ section override the corresponding configurations in the _typeRules_ section. The third section named _parameters_ affect global behavior.
+The configuration is specified in JSON format. It has two high-level sections. One of these sections, namely _fhirPathRules_ is meant to specify de-identification methods for data elements. The second section named _parameters_ affect global behavior.
 
 Here is a sample configuration:
 
 ```json
 {
-  "pathRules": {
-    "Patient.address.state": "keep",
-    "Patient.address.country": "redact"
-  },
-  "typeRules": {
-    "date": "dateShift",
-    "Address": "redact"
-  },
+  "fhirPathRules": [
+    {"path": "nodesByType('Extension')", "method": "redact"},
+    {"path": "Organization.identifier", "method": "keep"},
+    {"path": "nodesByType('Address').country", "method": "keep"},
+    {"path": "Group.name", "method": "redact"}
+  ],
   "parameters": {
     "dateShiftKey": "",
     "enablePartialAgesForRedact": true
 }
 ```
 
-### Path Rules
-Path rules are key-value pairs that can be used to specify the de-identification methods for individual elements. Ex:
+### FHIR Path Rules
+FHIR path rules can be used to specify the de-identification methods for individual elements. Ex:

```

Comment:
'for individual elements as well as elements of specific data types.' ?

Reply:
Updated. Thanks